src/Makefile-env.am

```diff
@@ -256,8 +256,6 @@ if ENABLE_COVERAGE
 EXTRALIBS += -lgcov
 endif # ENABLE_COVERAGE
 
-LIBCOMMON += -luuid
```

Comment:
@ukernel 
Right is seems that I'm able to compile and link without -luuid.
Now running tests, but the first 30 tests are fine